mendeleydesktop.appdata.xml

```diff
@@ -43,9 +43,15 @@
   <launchable type="desktop-id">mendeleydesktop.desktop</launchable>
 
   <screenshots>
-​    <screenshot type="default">
-​      <caption>The user interface</caption>
-      <image>https://2.bp.blogspot.com/-oZnl3C750Qg/TxmKMJgfdhI/AAAAAAAAHdM/LNwTZC-UE8o/s1600/mendeley-desktop_3.png</image>
+    <screenshot type="default">
+      <caption>User interface</caption>
+      <image>http://thefinalzone.net/img/apps/mendeley/mendeley-desktop_1.png</image>
```

Comment:
So you are really doing mendeley's job for them at this point. Do you have a financial interest in their success? 

 I really don't like two aspects of the screenshots:
1. They are hosted on a server I don't recognize. Is this your domain name?
2. They will surely become stale.
3. Nobody can ever see them because this package isn't on any repository.

Reply:
Someone was looking to get a rpm version of mendeley which led here. I read the documentation about the redistribution which I thank you for the effort. Basically, it has nothing to do with financial interest but to get things work.

 Those screenshots can be updated within the xml file at any time. I was unable to get the official version for the time being until someone will create a newer version.

1. Well aware and they are temporarily on my host until I can get a proper version and possibly talk either  with freedesktop team or mendeley.
2. That will change soon.
3. For now until this package will get one.